resources/session04/soup/test_mashup.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from mashup import result_generator, get_geojson, set_marker_color
+
+
+def test_result_generator():
+    result_list = []
+    for result in result_generator('Average Score', True, 5):
+        result_list.append(result)
+    for i in range(len(result_list)-1):
+        assert result_list[i]['Average Score'] >= result_list[i+1]['Average Score']
+    assert len(result_list) == 5
+    result_list = []
+    for result in result_generator('High Score', False, 5):
+        result_list.append(result)
+    for i in range(len(result_list)-1):
+        assert result_list[i]['High Score'] <= result_list[i+1]['High Score']
+    assert len(result_list) == 5
+    result_list = []
+    for result in result_generator('Total Inspections', True, 1):
+        result_list.append(result)
+    assert len(result_list) == 1
```

Comment:
In general it's a good idea when testing to make each test do as little as possible.  I'd suggest that breaking this test into three separate tests, one each for each of the calls to result_generator you make above.  The same goes for the test below as well.